adyenv6b2ccheckoutaddon/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -378,7 +378,7 @@ public PaymentsResponse handleRedirectPayload(final String payload) {
                 throw new InvalidCartException("Merchant reference doesn't match cart's code");
             }
 
-            if (PaymentsResponse.ResultCodeEnum.RECEIVED == response.getAuthResponse() || PaymentsResponse.ResultCodeEnum.AUTHORISED == response.getAuthResponse()) {
+            if (PaymentsResponse.ResultCodeEnum.RECEIVED == response.getResultCode() || PaymentsResponse.ResultCodeEnum.AUTHORISED == response.getResultCode()) {
```

Comment:
Have we tested it with openinvoice?

Reply:
openinvoice goes via api without redirects, so it shouldn't hit this logic.